auth-server/src/services/auth.ts

```diff
@@ -216,7 +217,19 @@ export default class AuthService {
 				email,
 				email_verified: false
 			})
-			.findById(Number(decoded.sub))
+			.findById(userId)
+
+		// Invalidate all refresh token for user
+		await RefreshToken
```

Comment:
I'm confused. We don't need to invalidate the refresh token for an email modification.

Reply:
not invalidating